example/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'java'
+apply plugin: "net.ltgt.apt"
+
+dependencies {
+    compile 'com.google.code.gson:gson:2.6.2'
+    compile 'com.google.auto.value:auto-value:1.2'
```

Comment:
Please update this to demonstrate the proper scoping of AutoValue using `apt` and `compileOnly`.